cluster-autoscaler/simulator/dynamicresources/utils/sanitize_test.go

```diff
@@ -416,6 +445,20 @@ func TestSanitizedPodResourceClaims(t *testing.T) {
 				TestClaimWithPodReservations(TestClaimWithAllocation(newOwnerClaim2, singleNodeAllocationSanitized), newOwner),
 			},
 		},
+		{
```

Comment:
Could you add test cases for:
1. Pod with multiple claims, one completely without admin access and one with
2. The case I mention in the other comment - A Pod with a ResourceClaim that's available on more than 1 Node, but all allocations are admin-access.